### DISCUSSION TITLE
Where do I find scripts?

### DISCUSSION THREAD

**Discussion by deroverda** [quoid/userscripts#489] [Q&A] [answered]
Hi. Total noob here. Just installed the extension and don't know where to go from here. Basically what I'm looking for is a dark mode script. Can someone please help me?

**Comment by quoid** [comment, marked as answer]
Here's a good basic tutorial around user scripts: https://wiki.greasespot.net/Greasemonkey_Manual

And here's an article about where you can find some: https://wiki.greasespot.net/User_Script_Hosting

Basic JS knowledge will also be beneficial: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide

Good luck!

**Reply by deroverda** [reply]
Forgot to reply, sry. Thank you so much!
